### ISSUE TITLE
Makefiles

### ISSUE THREAD

**Issue by rkdarst** [ResearchSoftwareHour/rsh-notes#9]
We can introduce the basics of a Makefile.  It's very primitive, and it may be better to demonstrate it for data automation than compiling, but a simple Makefile is very simple and universal.  I would see it as not primarily about Makefiles, but mainly a way to discuss automation without tech getting in the way and preparing for more advanced topics.

**Comment by bast** [comment]
I agree about showing this on a data processing example. Also an example where tasks can be run in parallel.

**Comment by wmvanvliet** [comment]
Shameless self promotion: https://blogs.aalto.fi/marijn/2016/02/19/make-intelligent-plumbing-for-your-analysis-pipeline/
